.github/workflows/publish-release.yml

```diff
@@ -3,7 +3,6 @@ name: Publish release
 on:
   release:
     types: [created]
-  workflow_dispatch:
```

Comment:
You don't think we want this on demand? How do you want to release the LTS images?

Reply:
> You don't think we want this on demand?
The workflow currently will error when triggered manually. LTS patches can be created by creating the corresponding release and NOT marking it as `latest` in the release page.

> How do you want to release the LTS images?
For this version I'll upload them manually.